themes/bootstrap3/templates/RecordDriver/SolrDefault/data-authors.phtml

```diff
@@ -1,15 +1,29 @@
 <?
-  $formatRoles = function ($roles) {
-    if (count($roles) == 0) {
-      return '';
+$formatProperty = function ($datafield, $name, $label) {
+    if (count($datafield) == 0) {
+        return '';
     }
     $that = $this;
-    $translate = function ($str) use ($that) {
-      return $that->transEsc('CreatorRoles::' . $str);
+    $translate = function ($str) use ($that, $label) {
+        return $that->transEsc($label . $str);
     };
-    return ' (' . implode(', ', array_unique(array_map($translate, $roles))) . ')';
-  };
+    return '<span class="author-property-' . $name . '">(' . implode(', ', array_unique(array_map($translate, $datafield))) . ')</span>';
+};
 ?>
 <? if (isset($data[$type]) && !empty($data[$type])): ?>
-  <? $i = 0; foreach ($data[$type] as $author => $roles): ?><?=($i++ == 0)?'':', '?><span property="<?=$this->escapeHtml($schemaLabel)?>"><a href="<?=$this->record($this->driver)->getLink('author', $author)?>"><?=$this->escapeHtml($author)?></a><?=$formatRoles($roles)?></span><? endforeach; ?>
+  <? $i = 0; foreach ($data[$type] as $author => $dataFields): ?>
+    <?=($i++ == 0)?'':', '?><div property="<?=$this->escapeHtml($schemaLabel)?>">
```

Comment:
There seem to be some layout problems here -- the reason the old code was all jammed into a long, hard-to-read line is that if you introduce whitespace, it causes spaces to appear in front of commas, which is not desirable. Also, it looks like some spans have been changed to divs, which is causing unwanted newlines in the list.